YACS-RCOS/yacs.n
# Prereq links

**Issue**

Closes #283, #302 

**Database Changes/Migrations**
N/A

**Test Modifications**
N/A

**Test Procedure**
1. Open any course page with prerequistes
2. Notice the links and distinctions between "and"s and "or"s

**Photos**
Before:
![Screen Shot 2020-11-13 at 3 49 18 PM](https://user-images.githubusercontent.com/59896953/99119805-d4922800-25c7-11eb-811c-35b1a14ca344.png)


After:
![Screen Shot 2020-11-13 at 3 48 11 PM](https://user-images.githubusercontent.com/59896953/99119735-af9db500-25c7-11eb-9570-8fc6a69b464c.png)


**Additional Info**
Also ended up deleting some more legacy code
